src/js/tech/tech.js

```diff
@@ -218,12 +218,13 @@ class Tech extends Component {
       return;
     }
 
+    let player = this.player();
     let textTracksChanges = function() {
       let updateDisplay = Fn.bind(this, function() {
-        this.trigger('texttrackchange');
+        player.trigger('texttrackchange');
```

Comment:
The event `texttrackchange` is triggered by the tech. Then, the player re-trigger it on himself if it is the active tech. I'm not sure what the problem is. Is it happening before the tech is loading?